cura/Snapshot.py

```diff
@@ -211,3 +230,8 @@ def snapshot(width = 300, height = 300):
             transformMode = QtCore.Qt.TransformationMode.SmoothTransformation)
 
         return scaled_image
+
+    @staticmethod
+    def _prereadSnapshot(snap = QImage):
```

Comment:
Why default the `snap` arg to a `QImage`? didn't you mean to type the function in this case?
```suggestion
    def _prereadSnapshot(snap: QImage) -> bool:
```